tackle-test-generator-unit/test/java/org/konveyor/tackle/testgen/TestgenIntegrationTest.java

```diff
@@ -117,7 +118,7 @@ public void testModelerInitializerExtender() throws Exception {
             // create building-block test sequences using combined test generator
             TestSequenceInitializer seqInitializer = new TestSequenceInitializer(testApp.appName,
                 testApp.testPlanFilename, testApp.appPath, testApp.appClasspathFilename,
-                Constants.COMBINED_TEST_GENERATOR_NAME, 5, false, false, "");
```

Comment:
The flaky tests are in the cli regression, not in the core regression. Please revert this change.

Reply:
This test was also failing consistently (I tried 3-4 runs); I can revert it if you'd prefer that

Reply:
Reverted